Shopify.md

```diff
@@ -0,0 +1,19 @@
+# BTCPay Server and Shopify Integration
+
+To integrate BTCPay Server into Shopify, you need to be running BTCPay Server v1.0.5.5 or newer.
+
+* In your BTCPay Server Store > Pay button enable `Allow anyone to create invoice`
```

Comment:
Probably not needed

Reply:
Yeah, @Kukks took care of this with new endpoint, no longer a requirement